src/components/ListItem/ListItem.tsx

```diff
@@ -30,44 +31,97 @@ export const ListItem: React.FC<ListItemProps> = ({
             tabIndex={tabIndex}
             {...props}
         >
-            {
-                iconStart && !image && (
-                    <div className="moonstone-listItem_iconStart">
-                        <iconStart.type {...iconStart.props} size={iconSize}/>
-                    </div>
-                )
-            }
+            <ConditionalWrapper
```

Comment:
I'm wondering if it will not be easier to maintain/read by adding a condition based on `image` to return two possible renders because we use `image` for almost all conditions.
I'm afraid to add a lot of complexity to this component.

Reply:
I don't get the point of this ConditionalWrapper at all .. ? it complexifies a lot and do not bring anything ?

Reply:
Fixed. 
I went to a lot of trouble for nothing, trying to avoid adding an unnecessary div if there was no description. But the conditionWrapper does make reading the code more complex than it should be. 
Thanks :-)